src/index.js

```diff
@@ -0,0 +1,148 @@
+const getTempFromCoordinates = () => {
+  const cityInput = document.getElementById('city-display').textContent;
+  console.log(cityInput);
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: cityInput,
+      },
+    })
+    .then((response) => {
+      console.log('success', response);
+      const return_obj = {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+      axios
+        .get('http://127.0.0.1:5000/weather', {
+          params: {
+            lat: return_obj['lat'],
+            lon: return_obj['lon'],
+          },
+        })
+        .then((weatherResponse) => {
+          console.log(weatherResponse);
+          const cityTemp = weatherResponse.data['main']['temp'];
+          let tempDisplay = document.getElementById('temperatureDisplay');
+          console.log(cityTemp);
+          const tempFahrenheit =
+            (parseInt(cityTemp) - parseInt(273.15)) * 1.8 + 32;
+          console.log(tempFahrenheit);
+          tempDisplay.textContent = tempFahrenheit;
+
+          changeTempColor(tempDisplay);
+          const garden = gardenChange(tempDisplay);
+
+          const gardenContainer = document.getElementById('garden_section');
+          gardenContainer.textContent = garden;
+        });
+    })
```

Comment:
Consider breaking this function up a bit. It's taking on a lot responsibilty. Whenever we see a function that is more than 10 lines long we should consider seprating concerns. This is completely preferential but just a note. More on this here:

https://dmitripavlutin.com/the-art-of-writing-small-and-plain-functions/

https://medium.com/@janakachathuranga/coding-best-practices-javascript-write-small-functions-7d2567bd6328

Reply:
There's an issue with wave 04 (calling the apis), the requirements specify that your code should get the weather of the currently displayed city name & when the  <button> element is clicked updates and displays the realtime temperature of the currently displayed city name. Currently your code is not providing the updated temperature.